### PR TITLE
Add test case of StandardTransactionContexts

### LINKED ISSUE #7148: Add test case for StandardTransactionContexts

### PR DESCRIPTION
**Pull request by xbkaishui**
Fixes #7148.

Changes proposed in this pull request:
- Add test case of StandardTransactionContexts

### CODE REVIEW

File: shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.apache.shardingsphere.transaction.ShardingTransactionManagerEngine;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+
+public class StandardTransactionContextsTest {
```
Comment (terrymanu) [inline shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java:30]: Please add final if the class is not design for extension

File: shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    
+    @Test
+    public void assertGetEnginesNull() {
```
Comment (terrymanu) [inline shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java:33]: What's mean of`assertGetEnginesNull`? Do you mean `assertNewInstanceWithEmptyEngines`?

File: shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        StandardTransactionContexts standardTransactionContexts = new StandardTransactionContexts();
+        Map<String, ShardingTransactionManagerEngine> engines = standardTransactionContexts.getEngines();
+        Assert.assertTrue(engines.isEmpty());
```
Comment (terrymanu) [inline shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java:36]: Please use static import with `Assert.assertTrue`

File: shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    public void assertGetEngines() {
+        Map<String, ShardingTransactionManagerEngine> actualEngine = new HashMap<>();
+        actualEngine.put(DefaultSchema.LOGIC_NAME, new ShardingTransactionManagerEngine());
+        StandardTransactionContexts standardTransactionContexts = new StandardTransactionContexts(actualEngine);
+        
+        Map<String, ShardingTransactionManagerEngine> engines = standardTransactionContexts.getEngines();
+        Assert.assertThat(engines.size(), is(1));
+        Assert.assertThat(engines, is(actualEngine));
+        ShardingTransactionManagerEngine defaultEngine = standardTransactionContexts.getDefaultTransactionManagerEngine();
+        Assert.assertNotNull(defaultEngine);
```
Comment (terrymanu) [inline shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java:49]: Please use static import with `Assert. assertNotNull`

File: shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    
+    @Test
+    public void assertGetEngines() {
+        Map<String, ShardingTransactionManagerEngine> actualEngine = new HashMap<>();
+        actualEngine.put(DefaultSchema.LOGIC_NAME, new ShardingTransactionManagerEngine());
+        StandardTransactionContexts standardTransactionContexts = new StandardTransactionContexts(actualEngine);
+        
+        Map<String, ShardingTransactionManagerEngine> engines = standardTransactionContexts.getEngines();
+        Assert.assertThat(engines.size(), is(1));
+        Assert.assertThat(engines, is(actualEngine));
```
Comment (terrymanu) [inline shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java:47]: Please use static import with `Assert. assertThat`

File: shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        Map<String, ShardingTransactionManagerEngine> engines = standardTransactionContexts.getEngines();
+        Assert.assertTrue(engines.isEmpty());
+    }
+    
+    @Test
+    public void assertGetEngines() {
+        Map<String, ShardingTransactionManagerEngine> actualEngine = new HashMap<>();
+        actualEngine.put(DefaultSchema.LOGIC_NAME, new ShardingTransactionManagerEngine());
+        StandardTransactionContexts standardTransactionContexts = new StandardTransactionContexts(actualEngine);
+
```
Comment (terrymanu) [inline shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java:44]: Please remove useless empty line

File: shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    
+    @Test
+    public void assertGetEnginesNull() {
+        StandardTransactionContexts standardTransactionContexts = new StandardTransactionContexts();
+        Map<String, ShardingTransactionManagerEngine> engines = standardTransactionContexts.getEngines();
+        Assert.assertTrue(engines.isEmpty());
+    }
+    
+    @Test
+    public void assertGetEngines() {
```
Comment (terrymanu) [inline shardingsphere-transaction/shardingsphere-transaction-core/src/test/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContextsTest.java:40]: It seems we do not need to assert method `getEngines` which is for simple getter only.
It is necessary to assert method `getDefaultTransactionManagerEngine`

**Review by terrymanu — CHANGES_REQUESTED** [review]
We are still missing assertion of `close` method.

**Comment by xbkaishui** [comment]
fix all the comments

**Review by terrymanu — APPROVED** [review]

**Pull request merged**